bolt-contracts/src/contracts/BoltSymbioticMiddleware.sol

```diff
@@ -0,0 +1,477 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import {Ownable} from "lib/openzeppelin-contracts/contracts/access/Ownable.sol";
+import {Time} from "lib/openzeppelin-contracts/contracts/utils/types/Time.sol";
+import {EnumerableMap} from "lib/openzeppelin-contracts/contracts/utils/structs/EnumerableMap.sol";
+import {EnumerableSet} from "lib/openzeppelin-contracts/contracts/utils/structs/EnumerableSet.sol";
+import {IERC20} from "lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+
+import {IBaseDelegator} from "@symbiotic/interfaces/delegator/IBaseDelegator.sol";
+import {Subnetwork} from "@symbiotic/contracts/libraries/Subnetwork.sol";
+import {IVault} from "@symbiotic/interfaces/vault/IVault.sol";
+import {IRegistry} from "@symbiotic/interfaces/common/IRegistry.sol";
+import {IOptInService} from "@symbiotic/interfaces/service/IOptInService.sol";
+import {ISlasher} from "@symbiotic/interfaces/slasher/ISlasher.sol";
+import {IVetoSlasher} from "@symbiotic/interfaces/slasher/IVetoSlasher.sol";
+import {IEntity} from "@symbiotic/interfaces/common/IEntity.sol";
+
+import {MapWithTimeData} from "../lib/MapWithTimeData.sol";
+import {IBoltValidators} from "../interfaces/IBoltValidators.sol";
+import {IBoltManager} from "../interfaces/IBoltManager.sol";
+import {IBoltMiddleware} from "../interfaces/IBoltMiddleware.sol";
+
+contract BoltSymbioticMiddleware is IBoltMiddleware, Ownable {
+    using EnumerableSet for EnumerableSet.AddressSet;
+    using EnumerableMap for EnumerableMap.AddressToUintMap;
+    using MapWithTimeData for EnumerableMap.AddressToUintMap;
+    using Subnetwork for address;
+
+    // ========= STORAGE =========
+
+    /// @notice Validators registry, where validators are registered via their
+    /// BLS pubkey and are assigned a sequence number.
+    IBoltValidators public validators;
+
+    /// @notice Set of Symbiotic operator addresses that have opted in to Bolt Protocol.
+    EnumerableMap.AddressToUintMap private symbioticOperators;
+
+    /// @notice Set of Symbiotic protocol vaults that are used in Bolt Protocol.
+    EnumerableMap.AddressToUintMap private symbioticVaults;
```

Comment:
now we can just call these `operators` and `vaults`.
the same with immutables below: we don't need SYMBIOTIC prefix anymore